migration/src/main/java/kin/sdk/migration/interfaces/MigrationManagerListener.java

```diff
@@ -0,0 +1,22 @@
+package kin.sdk.migration.interfaces;
+
+public interface MigrationManagerListener {
+
+    /**
+     * Method is invoked before the migration process itself will start.
+     */
+    void onMigrationStart();
+
+    /**
+     * Method is invoked when kinClient is ready to use.
```

Comment:
whether migration happened or not (final method either this or onError will be fired)